lib/credit_card_widget.dart

```diff
@@ -304,22 +307,10 @@ class _CreditCardWidgetState extends State<CreditCardWidget>
                       scale: 1,
                     ),
                   ),
-                const Spacer(),
-                Align(
-                  alignment: Alignment.topRight,
-                  child: Padding(
-                    padding: const EdgeInsets.only(left: 16, right: 16, top: 8),
-                    child: widget.cardType != null
-                        ? getCardTypeImage(widget.cardType)
-                        : getCardTypeIcon(widget.cardNumber),
-                  ),
-                ),
               ],
             ),
           ),
-          const SizedBox(
-            height: 10,
-          ),
+          const SizedBox(height: 10),
           Expanded(
             child: Padding(
               padding: const EdgeInsets.only(left: 16),
```

Comment:
Give horizontal padding here and remove padding from line number 326, I guess it will reflect same as expected.